src/cogniteClient.ts

```diff
@@ -291,12 +291,64 @@ export default class CogniteClient {
   public getMetadata = (value: any) => this.metadataMap.get(value);
 
   /**
-   * Do plain get-requests using the client
+   * Basic HTTP method for GET
+   *
+   * @param path The URL path
+   * @param options Request options, optional
+   *
+   * ```js
+   * import { CogniteClient } from '@cognite/sdk';
```

Comment:
I think you can drop all the boilerplate

Reply:
Okay so rather just do `const response = ...` and drop the rest?